md/widget-properties.md

```diff
@@ -76,6 +77,26 @@ By default, the [Bootstrap](http://getbootstrap.com/) CSS library is available s
 
 This property controls whether the widget is hidden. Bind this property to a variable. The value of the variable determines whether or not the widget is displayed. You can use this to control conditional display, for example when a widget becomes visible when the user enters a certain value in a form.
 
+### URL's
+
+When you need to reference a Bonita internal URL (an API call, a link to an other page / form ...), **do not hard code the webapp name. (eg: `/bonita` in a Development environment)**  
+You have the possibility to **use relative paths** in the widget properties / data.  
+
+So, instead of writing
+
+```
+http://localhost:8080/bonita/portal/resource/app/myApp/myPage/API/[...]
+```  
+
+You can simply write
+
+```
+../API/[...]
+```  
+
+Hard coding the webapp name ('_/bonita_') is a bad practice. It will work in the development environment, because the webapp name of the Bonita Studio tomcat is always _/bonita_, but on your production environment it is likely to change.  
```

Comment:
```suggestion
Hard coding the webapp name ('_/bonita_') is a bad practice. It will work in the development environment, because the webapp name of the Bonita Studio tomcat is always _bonita_, but on your production environment it is likely to change.  
```